bot/cogs/bot.py

```diff
@@ -60,6 +62,23 @@ def __init__(self, bot: AutoShardedBot):
 
         await ctx.invoke(self.info)
 
+    async def on_message(self, msg: Message):
+        if msg.content.count("\n") >= 3:
+            try:
+                tree = ast.parse(msg.content)
+                # Attempts to parse the message into an AST node.
+                # Invalid Python code will raise a SyntaxError.
+                if not all(isinstance(node, ast.Expr) for node in tree.body):
+                    # we dont want multiple lines of single words,
+                    # they would be syntactically valid Python but could also be
+                    # just some random multiline text someone is sending.
```

Comment:
This is pretty minor, but in future separate and capitalise sentences when commenting.
Refer to the comments above this one.

Reply:
the comment content itself is pretty good now though.